src/robot/reporting/jsmodelbuilders.py

```diff
@@ -22,9 +22,10 @@
 class JsModelBuilder(object):
 
     def __init__(self, log_path=None, split_log=False,
-                 prune_input_to_save_memory=False):
+                 prune_input_to_save_memory=False, auto_expand_keywords=None):
```

Comment:
The `prune_input_to_save_memory` is somewhat special argument. I'd keep it last even though it would possibly break backwards compatibility. It's very unlikely anyone calls this outside Robot's code and if they do, using `prune_input_to_save_memory=xxx` is a good idea anyway.

Reply:
done